package.json

```diff
@@ -10,6 +10,8 @@
     "temp-dir": "/tmp/enigma-p2p"
   },
   "scripts": {
+    "start": "node src/server.js",
```

Comment:
Nice addition!
Does it work with no args?

Reply:
yes. only env vars